integration-tests/emulated/src/tests/peregrine/did.rs

```diff
@@ -93,12 +93,13 @@ fn test_did_creation_from_asset_hub() {
 			]
 		);
 
-		// we also expect that the sovereignAccount of AssetHub has some coins now
-		let balance_after_xcm_call: u128 =
-			<<Peregrine as Parachain>::Balances as Inspect<AccountId>>::balance(&asset_hub_sovereign_account);
+		let balance_on_hold = <<Spiritnet as Parachain>::Balances as Inspect<AccountId>>::balance_on_hold(
+			&peregrine_runtime::RuntimeHoldReason::from(did::HoldReason::Deposit),
+			&asset_hub_sovereign_account,
+		);
 
-		// since a did is created some of the free balance should now be on hold. Therefore the balance should be less.
-		assert!(balance_after_xcm_call < init_balance);
+		// since a did is created, 2 of the free balance should now be on hold
+		assert_eq!(balance_on_hold, UNIT * 2);
```

Comment:
We can technically use the constant for the DID deposit to match against here.

Reply:
Sorry, I mixed up 2 commits. But yes thats a good hint: [d30d226](https://github.com/KILTprotocol/kilt-node/pull/603/commits/d30d22658eb2f0ba7c464b8d050d2222914a8da2)